l1-contracts/deploy-scripts/upgrade/EcosystemUpgrade.s.sol

```diff
@@ -881,6 +889,20 @@ contract EcosystemUpgrade is Script, DeployL1Script {
         });
     }
 
+    /// @notice Additional calls to newConfigure contracts
+    function prepareContractsConfigurationCalls() public virtual returns (Call[] memory calls) {
```

Comment:
please rename the function to `prepareDAValidatorCall`, currently the name is too vague